mod-data-import-converter-storage-server/src/main/java/org/folio/services/association/ProfileAssociationService.java

```diff
@@ -49,4 +48,24 @@
    * @return future with true if succeeded
    */
   Future<Boolean> delete(String id, String tenantId);
+
+  /**
+   * Finds details by master id.
+   *
+   * @param tenantId   a tenant id
+   * @param masterId   a master id
+   * @param masterType a master type
+   * @return list of details for specified master
+   */
+  Future<Optional<ProfileSnapshotWrapper>> findDetails(String tenantId, String masterId, ContentType masterType);
+
+  /**
+   * Finds masters by detail id.
+   *
+   * @param tenantId   a tenant id
+   * @param detailId   a detail id
+   * @param detailType a detail type
+   * @return list of masters profiles for specified detail profile
+   */
+  Future<Optional<ProfileSnapshotWrapper>> findMasters(String tenantId, String detailId, ContentType detailType);
```

Comment:
Please move 'tenantId' argument at the end of method arguments list

Reply:
Done